.kube-linter.yaml

```diff
@@ -0,0 +1,6 @@
+checks:
+  ignorePaths:
+    - .github/
+    - .aicoe-ci.yaml
```

Comment:
Why do we have the same file twice?

In fact... it looks like we don't use it or the `.thoth.yaml` much at all. Should we remove all 3 of these from the repo? @LaVLaS

Reply:
I must have overlooked it when I was changing the config file at one point, I removed the duplicate file mention.

Reply:
Correct. `.aicoe-ci.yaml` and `.thoth.yaml` can be removed.  Can you add `tests/` to the list  also